docs/uglify-options.md

```diff
@@ -7,113 +7,94 @@ This task primarily delegates to [UglifyJS2][], so please consider the [UglifyJS
 
 
 ## mangle
-Type: `Boolean` `Object`  
```

Comment:
Sorry about that - my editor was configured to automatically remove them.

I'll go through the changes and restore them.